src/nvim/CMakeLists.txt

```diff
@@ -80,6 +80,20 @@ endforeach()
 set_source_files_properties(
   ${CONV_SOURCES} PROPERTIES COMPILE_FLAGS "${COMPILE_FLAGS} -Wno-conversion")
 
+if(DEFINED MIN_LOG_LEVEL)
+  if(NOT MIN_LOG_LEVEL MATCHES "^[0-3]$")
+    message(FATAL_ERROR "MIN_LOG_LEVEL must be a number DEBUG(0), INFO(1), WARNING(2), ERROR(3)")
+  endif()
+  message(STATUS "Log level is set to ${MIN_LOG_LEVEL}")
+  add_definitions(-DMIN_LOG_LEVEL=${MIN_LOG_LEVEL})
+else()
+  # By default INFO(1), disabled for Release builds
+  if(NOT CMAKE_BUILD_TYPE STREQUAL "Release")
+    add_definitions(-DMIN_LOG_LEVEL=1)
+    message(STATUS "Log level not specified, defaulting to INFO")
+  endif()
```

Comment:
I assumed they would be disabled for Release - and log.h has a check for NDEBUG that should cover that case. I didn't want to default to risk a case where logs would be unintentionally disabled.